sio-digital-team/web-ui-sdk
# Modal functionality

https://trello.com/c/r1BpwMIr


**Changes:**

- [x] Added Modal JS,HTML,SCSS
- [x] Updated style.scss to include new dependencies
- [x] Modal functionality built with 3 size options
  

**How to test:**
- Define a trigger for the modal with `data-modal="#myModal"`
- Create modal based on the structure outlined in the wiki [here](https://github.com/sio-digital-team/web-ui-sdk/wiki/Components#modal).
- See if modal opens 
- Modal closes when clicking on the backdrop or any element that has the `data-close` attribute.
  

**Todo:**
- [x] Modal documentation
